bin/github-review.sh

```diff
@@ -38,44 +34,27 @@ if [[ "$#" -ne 1 ]]; then
 fi
 if ! [[ $1 =~ ^[0-9]+$ ]]; then
     echo "ERROR: $1 is not a valid GitHub PR number" 1>&2
-    usage
     exit 1
 fi
 pr=$1
-prChainID="$(gh pr view $pr | sed -nE 's/^- Chain_ID: ([0-9]+).*$/\1/p')"
-if [[ -z $prChainID ]]; then
+chainid="$(gh pr view $pr | sed -nE 's/^- Chain_ID: ([0-9]+).*$/\1/p')"
+if [[ -z $chainid ]]; then
     echo "ERROR: Chain ID not specified as per the PR Template" 1>&2
-    usage
-    exit 1
-fi
-if ! [[ $prChainID =~ ^[0-9]+$ ]]; then
-    echo "ERROR: $prChainID is not a valid Chain ID number" 1>&2
-    usage
-    exit 1
-fi
```

Comment:
This check is tautological, since the `chainid` value is read to match this pattern.